ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -137,6 +137,11 @@ vsphere)
 openstack*)
     # shellcheck disable=SC1090
     source "${SHARED_DIR}/cinder_credentials.sh"
+    if [[ "${CONFIG_TYPE}" == "proxy" ]]; then
```

Comment:
I'm not an OpenStack maintainer, but using `CONFIG_TYPE` for `proxy` seems like overloading.  What if you want to do single-node proxy?  What if you want to do compact proxy?  Can you switch on `test -n "${HTTP_PROXY}" -o "${HTTPS_PROXY}"` instead, with those being set from `${SHARED_DIR}/proxy-conf.sh` up around line 33?

Reply:
very good point. Thanks